### PR TITLE
Add Bedrock/1.19.63 (Protocol 568)

### PR DESCRIPTION
**Pull request by stevarino**
Looks like just a version bump to protocol 568.

### CODE REVIEW

**Comment by extremeheat** [comment]
If it's just a version bump (no changes to protocol) we don't need new protocol files, adding to the dataPath and protocolVersions.json file should work. The bump was expected in #682

File: data/bedrock/latest/proto.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Created from MiNET and gophertunnel docs
 # The version below is the latest version this protocol schema was updated for.
 # The output protocol.json will be in the folder for the version
-!version: 1.19.63
+!version: 1.19.62
```
Comment (stevarino) [inline data/bedrock/latest/proto.yml:4]: Just curious, what's the logic about reverting this field? Is it used anywhere or just git log cleanliness?

Comment (extremeheat) [reply data/bedrock/latest/proto.yml:4]: It's used when determining where the protocol.json file will be written (which is ultimately used by protodef)

**Review by extremeheat — APPROVED** [review]

**Comment by extremeheat** [comment]
Thanks!

**Pull request merged**